src/appengine/libs/handler.py

```diff
@@ -153,16 +153,29 @@ def wrapper(self):
   return wrapper
 
 
-def get_access_token(verification_code):
+# TODO(mbarbella): Remove the |use_reproduce_tool_client| argument and related
+# plumbing once clusterfuzz-tools is removed. We need to support both while the
```

Comment:
We can just use same client id and secret for the new version and rename the attributes names and set their values when deployed. Since it is only used on internal, once deployed, you can instantly change the values.
s/clusterfuzz_tools_oauth_client_id/reproduce_tool_client_id
s/clusterfuzz_tools_client_secret/reproduce_tool_client_secret
remove use_reproduce_tool_client arg completely from all functions
deploy on staging to set reproduce_tool_client_id and reproduce_tool_client_secret manually. i am even thinking to put both client id and secret in configuration as it feels weird to put one in a yaml and other in admin config, so lets move both to admin config.

Reply:
SGTM. For now I'll just reuse the clusterfuzz-tools client id/secret, and at some point in the future I'll switch it over.